src/main/java/org/geysermc/discordbot/listeners/BadLinksHandler.java

```diff
@@ -116,7 +116,7 @@ public void onMessageReceived(@NotNull MessageReceivedEvent event) {
 
             if (foundMatch) {
                 ServerSettings.getLogChannel(event.getGuild()).sendMessageEmbeds(new EmbedBuilder()
-                        .setAuthor(event.getAuthor().getAsTag(), null, event.getAuthor().getAvatarUrl())
+                        .setAuthor(event.getAuthor().getName(), null, event.getAuthor().getAvatarUrl())
```

Comment:
Also out of scope, I know you mentioned this in the description, but still. I'd rather replace all these and handle tags/usernames properly outside of this PR.